docs/development.md

```diff
@@ -36,3 +36,25 @@ build -c opt --define MEDIAPIPE_DISABLE_GPU=1 mediapipe/examples/desktop/object_
 'bazel-bin/mediapipe/examples/desktop/object_detection/object_detection_openvino --calculator_graph_config_file mediapipe/graphs/object_detection/object_detection_desktop_openvino_graph.pbtxt --input_side_packets "input_video_path=/mediapipe/mediapipe/examples/desktop/object_detection/test_video.mp4,output_video_path=/mediapipe/tested_video.mp4"'
 ```
 - in object_detection example the input video will get all the objects detected by the model and return the tested_video.mp4 output.
+
+# Setting log level
+
+Openvino Model Server has its own log_level setting that can be set to the following levels:
```

Comment:
I think it should be clearer to state that the log level in mediapipe application can be set using GLOG_minloglevel env variable. `0` is the most verbose for debuging purposes and 3 is the least verbose with the major errors only.
That setting is applicable both for mediapipe framework logs and also for OpenVINO inference calculator.